synapse/cortex.py

```diff
@@ -118,21 +127,30 @@ def count(self, text, opts=None):
         query = self._getStormQuery(text, opts=opts)
         return sum((1 for n in query.evaluate()))
 
-    def eval(self, text, opts=None):
+    def eval(self, text, opts=None, iden=None):
```

Comment:
Should iden be passed in? Or would it be better to generate it and return something else?

Reply:
If we like the way this works, should we implement something similar for `eval` on `Cortex`?

Reply:
This is something we should track on the Cortex itself, since a local user may have a query they need to cancel.